en/scripting/subpackage.md

```diff
@@ -1,49 +1,49 @@
-# 代码分包加载
+# Code Sub-packaging
 
-随着游戏玩法越来越丰富，游戏的代码量也越来越大，开发者对于扩大包大小的需求越来越强烈，同时微信小游戏也支持了分包加载的功能，所以 Cocos Creator 推出了代码分包加载这样一个功能。
+As gameplay becomes more and more rich, the amount of code in the game is increasing and developers are wanting to expand the size of their game. **Code sub-packaging** provides for these demands.
 
-分包加载，即把游戏内容按一定规则拆分在几个包里，在首次启动的时候只下载必要的包，这个必要的包称为 **主包**，开发者可以在主包内触发下载其他子包，这样可以有效降低首次启动的消耗时间。
+**Sub-package** loading, is the the game content being split into several packages according to certain rules. During the first startup, only the necessary packages are downloaded, also called the **main package**. Other sub-packages are downloaded, as triggered, which can effectively reduce the time spent on the first boot.
 
-## 配置方法
+## Configuration method
 
-Cocos Creator 的分包是以文件夹为单位来配置的，当我们选中一个文件夹时，在 **属性检查器** 中会出现文件夹的相关配置选项：
+__Cocos Creator's__ sub-package is configured in folders. When a folder is selected, the relevant configuration options for the folder appear in the **Property Checker**:
 
 ![subpackage](./subpackage/subpackage.png)
 
-勾选 **配置为子包** 后，点击右上方的 `应用`，这个文件夹下的代码就会被当做是子包的内容了。子包名会在加载子包时作为加载的名字传入，默认会使用这个文件夹的名字。
+After checking **Config as sub-package**, click `Apply` at the top right, and the code in this folder will be treated as a __sub-package__. The __sub-package__ name will be passed as the loaded name when the __sub-package__ is loaded. The name of this folder will be used by default.
```

Comment:
1. Config as sub-package -> Subpackage
![image](https://user-images.githubusercontent.com/19702899/48823802-20c17d80-ed9d-11e8-97ae-c9bd778a3bc8.png)
2. ...will be treated as a sub-package -> will be treated as **the contents of** the sub-package